File: Week 1 - C/Mario/Mario.c

#include <cs50.h>

#include <stdio.h>


int main(void) {
    //user input
    int length = get_int("Height: ");

    //main output
    for (int i = 0; i < length; i++) {
        int x = 0;
        while(x < 8-i) {
            printf(" ");
            x++;
        }
        for(int j = 0; j < i+1; j++) {
            printf("#");
        }
        printf("  ");
        for(int j = 0; j < i+1; j++) {
            printf("#");
        }
        printf("\n");
    }
}

File: Week 6 - Python/Mario/mario.py
length = int(input("Height: "))

while length > 8 or length < 1:
    length = int(input("Height: "))
    
for i in range(length+1):
    if i != 0 :
        print(" " * (length-i), end = "")
        print("#" * i)
